drivers/include/periph/gpio.h

```diff
@@ -224,27 +249,39 @@ int gpio_read(gpio_t pin);
 /**
  * @brief   Set the given pin to HIGH
  *
+ * @pre     The given pin has been configured as Output using
```

Comment:
these seem now incomplete. I can set a pin to "high" that has been initialized using `gpio_init_low/high()`

Reply:
Well of course :wink: